hoodie-common/src/main/avro/HoodieCommitMetadata.avsc

```diff
@@ -0,0 +1,61 @@
+{
+   "namespace":"com.uber.hoodie.avro.model",
+   "type":"record",
+   "name":"HoodieCommitMetadata",
```

Comment:
Looks good - do you want to add the fields as optional - which will make sure we can read old serialized commits with this schema